README.md

```diff
@@ -1,13 +1,45 @@
-# Shopify App CLI 
+# [Shopify App CLI](https://shopify.dev/tools/cli)
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md)[![Build Status](https://travis-ci.com/Shopify/shopify-app-cli.svg?token=qtPazgjyosjEEgxgq7VZ&branch=master)](https://travis-ci.com/Shopify/shopify-app-cli)
 
-Shopify App CLI helps you build Shopify apps faster. It automates many common tasks in the development process and lets you quickly add popular features, such as billing and webhooks.
+Shopify App CLI helps you build Shopify apps faster. It quickly scaffolds Node.js and Ruby on Rails apps and automates many common development tasks.
 
-## Using the CLI
-Consult the [Shopify App CLI documentation](https://shopify.github.io/shopify-app-cli) to get started.
+## Features
 
-## Developing the CLI
-This is an [open-source](https://github.com/Shopify/shopify-app-cli/blob/master/.github/LICENSE.md) tool and developers are [invited to contribute](https://github.com/Shopify/shopify-app-cli/blob/master/.github/CONTRIBUTING.md) to it. Please check the [code of conduct](https://github.com/Shopify/shopify-app-cli/blob/master/.github/CODE_OF_CONDUCT.md) before you begin.
+The Shopify App CLI helps your app development process in the following ways:
 
-[Read the development guide](https://github.com/Shopify/shopify-app-cli/wiki) to learn more.
+- Creates an app in the [Partner dashboard](https://partners.shopify.com/current/resources)
+- Populates test data for your app, including products, customers, and draft orders
+- Generates new [webhooks](https://shopify.dev/tutorials/manage-webhooks) for your app
+- Deploys your app to [Heroku](https://www.heroku.com/)
```

Comment:
Does this perhaps suggest that we're forcing people to use Heroku?

Reply:
It might! Heroku is the only option at the moment, but I can soften the statement to something like: 

> - Deploys your app to an external platform, such as Heroku

Reply:
👍